package.json

```diff
@@ -8,7 +8,8 @@
   ],
   "license": "BSD-4-Clause",
   "scripts": {
-    "build": "tsc"
+    "build": "tsc",
+    "generate-latest-types": "mkdir -p out && ts-node scripts/generate-latest-types.ts > out/latest-types"
```

Comment:
Could this be shortened to `generate-types`?

Reply:
This is meant to generate only the latest types. If we integrate something like what Albi was suggesting above, meaning generating all types for all runtimes, then I agree it would make sense to call it simply `generate-types`.